### PR TITLE
Clear up documentation of cl_dummy_control

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/game/variables.h [hunk 1]
```diff
@@ -138,10 +138,10 @@ MACRO_CONFIG_INT(ClDummyRestoreWeapon, cl_dummy_restore_weapon, 1, 0, 1, CFGFLAG
 MACRO_CONFIG_INT(ClDummyCopyMoves, cl_dummy_copy_moves, 0, 0, 1, CFGFLAG_CLIENT, "Whether dummy should copy your moves")
 
 // more controlable dummy command
-MACRO_CONFIG_INT(ClDummyControl, cl_dummy_control, 0, 0, 1, CFGFLAG_CLIENT, "Whether can you control dummy at the same time")
-MACRO_CONFIG_INT(ClDummyJump, cl_dummy_jump, 0, 0, 1, CFGFLAG_CLIENT, "Whether dummy is jumping")
-MACRO_CONFIG_INT(ClDummyFire, cl_dummy_fire, 0, 0, 1, CFGFLAG_CLIENT, "Whether dummy is firing")
-MACRO_CONFIG_INT(ClDummyHook, cl_dummy_hook, 0, 0, 1, CFGFLAG_CLIENT, "Whether dummy is hooking")
+MACRO_CONFIG_INT(ClDummyControl, cl_dummy_control, 0, 0, 1, CFGFLAG_CLIENT, "Whether can you control dummy at the same time (cl_dummy_jump, cl_dummy_fire, cl_dummy_hook)")
+MACRO_CONFIG_INT(ClDummyJump, cl_dummy_jump, 0, 0, 1, CFGFLAG_CLIENT, "Whether dummy is jumping (requires cl_dummy_control 1)")
```
Comment (BannZay) [inline src/game/variables.h:142]: dummy copies fire?
From what I know dummy is "your second tee" but not "one of your tee`s that is not active at the moment"
Am i wrong

**Comment by BannZay** [comment]
Not really a subject for this PR but I treat this PR as a warning that something wrong with this.
"cl_dummy_fire" should be better to call cl_dummy_control_fire or cl_dummy_copy_fire
cl_control is not respected by cl_dummy_copy_moves which makes it even harder to understand

Just talking from a user perspective, have not seen any code around this feature

**Comment by BannZay** [comment]
And yea, names are very important, correct console suggestions are way better than overloaded descriptions

File: src/game/variables.h (hunk 1 above)
Comment (BannZay) [inline src/game/variables.h:142]: Might more clear with "Whether dummy is copying jumping"

Comment (def-) [reply src/game/variables.h:142]: but that's not what the option is doing. You can actually separately make dummy jump without copying.

Comment (BannZay) [reply src/game/variables.h:142]: You are right, misunderstood the feature

**Pull request merged**